gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.10-bin.zip
```

Comment:
I guess this will be rolled back to 8.8, right? cc. @achmelo

Reply:
Andrej said it's fine, we can keep it at the latest version